docs/index.rst

```diff
@@ -4,24 +4,30 @@
     contain the root `toctree` directive.
 
 .. toctree::
-    :caption: Further documentation
     :hidden:
     :maxdepth: 3
 
-    contributing
-    Code Of Conduct <https://github.com/pypa/.github/blob/main/CODE_OF_CONDUCT.md>
     changelog
-    Python Packaging User Guide <https://packaging.python.org/tutorials/distributing-packages/>
-
-Welcome to twine's documentation!
-=================================
+    contributing
+    Code of Conduct <https://github.com/pypa/.github/blob/main/CODE_OF_CONDUCT.md>
```

Comment:
Did you consider having a separate ToC section for contributing-related items?

Reply:
I haven't. Can you give an example?

Reply:
Just have several separate toctrees with captions:
https://github.com/ansible/pylibssh/blob/devel/docs/index.rst / https://ansible-pylibssh.rtfd.io

Reply:
Ah, looks like that would mean splitting up `contributing.rst` into multiple documents. Maybe in the future, though the current format doesn't bother me.

Reply:
> Ah, looks like that would mean splitting up `contributing.rst` into multiple documents.

Not necessarily. I'd keep the document + external links like CoC in a new section even w/o splitting the contributing guide.